### PR TITLE
Build Windows wheels on GHA

### PR DESCRIPTION
**Pull request by dataflake**
Also see https://github.com/zopefoundation/meta/issues/235

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -243,4 +243,4 @@
 
 
 # Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {'http://docs.python.org/': None}
+intersphinx_mapping = {'python': ('https://docs.python.org/3', None)}
```
Comment (tseaver) [inline docs/conf.py:246]: Ugh, I've just tweaked that file to add additional references in #52.  Are we really sure we want to standardize `docs/conf.py` across repositories?

Comment (tseaver) [reply docs/conf.py:246]: Or maybe you added that change manually (I don't see a template for it in the `meta/config/c-code` tree, anyway) to silence the warning?  In which case I will just resolve the conflict.

**Review by tseaver — APPROVED** [review]
I looked further, and couldn't find a sign that `docs/config.py` was being overwritten by the `meta` tools, so..... LGTM.

**Comment by dataflake** [comment]
> I looked further, and couldn't find a sign that `docs/config.py` was being overwritten by the `meta` tools, so..... LGTM.

I made that change manually in reaction to a warning message during the tox run.

**Pull request merged**